config/install/payment.payment_method_configuration.payment_postfinance_payment_form.yml

```diff
@@ -1,13 +1,13 @@
-id: postfinance_payment_form
+id: payment_postfinance_payment_form
 label: 'Postfinance Payment Form'
 ownerId: 1
 pluginConfiguration:
-  message_text: ''
+  message_text: 'Postfinance Payment Form'
   message_text_format: plain_text
-  pspid: drupalDEMO
+  pspid: 'drupalDEMO'
```

Comment:
We should not have our test user in here, or any of the in/out keys. The default values should be empty.